### ISSUE TITLE
An issue with 'Branches aren't just for birds' challenge.

### ISSUE THREAD

**Issue by Kalinaki** [freeCodeCamp/freeCodeCamp#13690]
Challenge [Save your Code Revisions Forever with Git](https://www.freecodecamp.com/challenges/save-your-code-revisions-forever-with-git) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

1.From my understanding there is a conflict between requirement 1 and requirement 2 and 3. You are asked to create a branch with a name that matches your name on GitHub. Once this is done requirements 2 and 3 are fulfilled but requirement 1 stays undone. It asks that you create a branch with a name add-<Myname>...or in my case add-Kalinaki.
2. Being unable to resolve this issue leads to problems with completing "Requesting you Pull, Please" and "Merge Tada!"

I saw the guideline and renamed my branch few times while redoing the previous steps.
Some direction or tips would be nice.

One peace.

**Comment by raisedadead** [comment]
@Kalinaki thanks for the issue.

[`git-it`](https://github.com/jlord/git-it) has ended active support.

/cc @freeCodeCamp/moderators

**Comment by erictleung** [comment]
@raisedadead should we update this challenge to make use of the new updated Electron-ified version of the repository https://github.com/jlord/git-it-electron? I haven't tested it but it looks like they are supporting that over the original `git-it` repository.

**Comment by raisedadead** [comment]
@erictleung yup, that was my initial thought. Someone will have to try out the app and go thru the challenge instructions

Tagging this as help wanted.

**Comment by Manish-Giri** [comment]
@raisedadead Can I take this? I've already done the newer version.

**Comment by raisedadead** [comment]
@Manish-Giri sure.

**Comment by systimotic** [comment]
Would this be an update for backup/staging? It doesn't seem like the beta has git tutorials anymore.

**Comment by Manish-Giri** [comment]
@systimotic @raisedadead I started working on this and noticed that the [npm package `git-it`](https://www.npmjs.com/package/git-it) is the older, terminal based version. The newer version is the desktop app that needs to be downloaded and installed.

Since the [current challenge](https://www.freecodecamp.com/challenges/save-your-code-revisions-forever-with-git) has instructions to setup a workspace in c9 and run the older `git-it` there, should I rewrite the instructions completely with respect to the new app instead? I mean, to begin with, use the desktop app instead of c9, and so on.

**Comment by raisedadead** [comment]
> should I rewrite the instructions completely with respect to the new app instead?

Yes @Manish-Giri if you have done that, locally with the app, and it seems easy enough for you, we would love to move away to the app.
